### PR TITLE
chore(docker): upgrade to NPM7

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes:

- https://github.com/microsoft/playwright/issues/7867#issuecomment-889837085
- https://github.com/microsoft/playwright/issues/8678
- https://github.com/microsoft/playwright/issues/9043#issuecomment-923929585

### CODE REVIEW

File: utils/docker/Dockerfile.bionic [hunk 1]
```diff
@@ -7,6 +7,9 @@ RUN apt-get update && apt-get install -y curl && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
     apt-get install -y nodejs
 
+# Upgrade to NPM7 (https://github.com/microsoft/playwright/issues/7867#issuecomment-889837085)
+RUN npm install -g npm@latest
```
Comment (aslushnikov) [inline utils/docker/Dockerfile.bionic:11]: npm@7 ?

File: utils/docker/Dockerfile.bionic [hunk 2]
```diff
@@ -7,6 +7,11 @@ RUN apt-get update && apt-get install -y curl && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
     apt-get install -y nodejs
 
+# Upgrade to NPM7 (https://github.com/microsoft/playwright/issues/7867#issuecomment-889837085)
+RUN npm install -g npm@7 && \
+    # Cache folder contains root-owned files, due to a bug in previous versions of npm which has since been addressed.
+    chown -R 1001:121 "/root/.npm"
```
Comment (aslushnikov) [inline utils/docker/Dockerfile.bionic:13]: what's 121 group?

File: utils/docker/Dockerfile.bionic [hunk 3]
```diff
@@ -7,6 +7,11 @@ RUN apt-get update && apt-get install -y curl && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
     apt-get install -y nodejs
 
+# Upgrade to NPM7 (https://github.com/microsoft/playwright/issues/7867#issuecomment-889837085)
```
Comment (aslushnikov) [inline utils/docker/Dockerfile.bionic:10]: what's the URL? Why is it relevant to the NPM7 upgrade?

**Comment by weyert** [comment]
Please note that Yarn and NPM v7 aren't supported on Gitlab CI when using their NPM private registry

**Comment by mxschmitt** [comment]
> Please note that Yarn and NPM v7 aren't supported on Gitlab CI when using their NPM private registry. If someone is pushing `playwright` to their private registry and then try to use it they won't be able to use `playwright`.
> 
> See: https://gitlab.com/gitlab-org/gitlab/-/issues/327910 and https://gitlab.com/gitlab-org/gitlab/-/issues/275992

You should be still able to downgrade by using `npm i -g npm@7` until the GitLab team has fixed their compatiblity issues. Would be also awesome if you could test **after** this change has been merged on the next tag:

`docker pull mcr.microsoft.com/playwright:next-focal`

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**